content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Install Istio Multicluster on Google Kubernetes Engine clusters
+description: Example multicluster GKE install of Istio.
+weight: 65
+keywords: [kubernetes,multicluster]
+---
+
+This is an example deployment of multicluster Istio on 2
+[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) clusters.  It is an example
+using the [Istio kubernetes multicluster install instructions](/docs/setup/kubernetes/multicluster-install/).
+
+## Before you begin
+
+In addition to the prerequisites for installing Istio the following setup is required for this example:
+
+*  This sample requires a valid Google Cloud Platform project with billing enabled. If you are
+   not an existing GCP user, you may be able to enroll for a $300 US [Free Trial](https://cloud.google.com/free/) credit.
+
+   *  [Create a Google Cloud Project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) to
```

Comment:
remove the two leading spaces - so the bullet list is a list.

Reply:
Ok... yeah, there's some dance with the linter that I need to learn.  The list/bullets containing `{{< text bash >}}` seem to require 2 spaces but if you have some bullets/list items with different number of spaces I think the linter complains.